roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1077,6 +1077,11 @@ def get_defaults(self, roles):
                         iptables_sync_period='5s', set_node_ip=False)
             defaults['node'] = node
 
+        if 'nfs' in roles:
+            nfs = dict(exports_dir='/var/export', registry_volume='regvol',
+                       export_options='*(rw,sync,all_squash)')
```

Comment:
I thought I saw somewhere that all_squash wasn't needed anymore...  might want to check with @markturansky